scripts/process/scaleStates.R

```diff
@@ -0,0 +1,41 @@
+#scale states according to water use
+# two different sized states with equal water use should have the 
+# same area
+library(rgeos)
+library(dplyr)
+
+process.scaleStates <- function(viz){
+  #could put these in yaml too?
+  coreCats <- c("Thermoelectric", "Irrigation", "Public Supply", "Industrial")
+  
+  #read sp, wuClean
+  statePoly <- readData(viz[['depends']]$stateMap)
+  wuClean <- readData(viz[['depends']]$wuClean)
+  
+  wuClean$state_name <- tolower(wuClean$state_name)
+  #get areas, join to wuClean
+  areas <- data.frame(gArea(statePoly, byid = TRUE), stringsAsFactors = FALSE)
+  areas$state_name <- rownames(areas)
+  areas["district of columbia",][2] <- "dist. of columbia"
+  wuAreas <- left_join(wuClean, areas, by = 'state_name')
+  names(wuAreas)[6] <- "area"
+  
+  #compute ratios
+  wuAreas <- mutate(wuAreas, wuPerArea = value/area)
+  finalScaleFactors <- data.frame()
+  #per category
+  for(cat in coreCats){
+    thisCat <- filter(wuAreas, category == cat)
+    maxRatio <- max(thisCat$wuPerArea, na.rm = TRUE)
+    thisCat <- mutate(thisCat, scale = wuPerArea/maxRatio)
+    thisCat <- mutate(thisCat, newArea = area*scale)
+    thisCat <- mutate(thisCat, scaleFactor = newArea/area)
+    maxScaleFactor <- range(thisCat$scaleFactor, na.rm = TRUE, finite = TRUE)[2]
+    thisCat <- mutate(thisCat, scaleFactor = scaleFactor/maxScaleFactor)
```

Comment:
have you run into cases where maxScaleFactor differs from 1? I can't think of a time when I'd expect that, given that you're dividing all the areas by the maximum area, and one of the areas == that maximum area (such that scale=1 for at least one state), and none of the areas > maximum areas (such that scale never exceeds 1 for any state).

unless i'm just failing to imagine some scenario, i think `scale` and `scaleFactor` will always be identical, and so you might as well not calculate scaleFactor or maxScaleFactor.